analyzers/tests/SonarAnalyzer.UnitTest/Helpers/KnownTypeTest.cs

```diff
@@ -0,0 +1,18 @@
+﻿using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SonarAnalyzer.Helpers;
+
+namespace SonarAnalyzer.UnitTest.Helpers
+{
+    [TestClass]
+    public class KnownTypeTest
```

Comment:
what if the KnownType isn't an Attribute? can you add a test for "X" - it will throw...

Reply:
I've added a test for this in `AttributeSyntaxExtensionsTest`.